src/defaultValues.js

```diff
@@ -0,0 +1,16 @@
+require('dotenv').config();
+
+const DEFAULT_PORT   = 8000;
+const DEFAULT_ADDRESS = 'localhost';
+const maxItems = 28;
+const key = process.env.ACCESS_KEY;
+const token = process.env.ACCESS_TOKEN;
+module.exports = {
```

Comment:
We add an empty line before the first line of a multiline expression and after its last line. Compare:
```js
const key = process.env.ACCESS_KEY;
const token = process.env.ACCESS_TOKEN;

module.exports = {
    //foo
    //bar
};

function foo () {
 //foo
}
````